templates/route.tpl

```diff
@@ -1 +1 @@
-Route::apiResource('%base_endpoint%', '%controller_class%', [%options%]);
+Route::%router_method%('%base_endpoint%', '%controller_class%')%middleware_string%;
```

Comment:
Let's still use one template:
```suggestion
Route::%router_method%('%base_endpoint%', '%controller_class%'%options%)%route_methods%;
```
`%options%` and `%route_methods%` are optional based on the `%router_method%` used.